components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/OptimizedAuthenticatorConfig.java

```diff
@@ -45,6 +45,7 @@ public class OptimizedAuthenticatorConfig implements Serializable {
     private final AuthenticatorStateInfo authenticatorStateInfo;
     private final Map<String, String> parameterMap;
     private final List<String> idPResourceIds;
+    private String tenantDomain;
```

Comment:
this can be final

Reply:
Resolved with [`646a5dd`](https://github.com/wso2/carbon-identity-framework/pull/4446/commits/646a5dd851a1d454c0d80813f913fc7f4beb71d7)